packages/amplify-provider-awscloudformation/lib/build-resources.js

```diff
@@ -1,4 +1,5 @@
 const fs = require('fs');
+const fsExtra = require('fs-extra');
```

Comment:
Shouldn't we just use fs-extra and remove the fs import from the line above? fs-extra has all the fn's that are present in fs I beleive.